integration_tests/cairozero_test.go

```diff
@@ -19,8 +20,16 @@ func TestCairoZeroFiles(t *testing.T) {
 	testFiles, err := os.ReadDir(root)
 	require.NoError(t, err)
 
+	// Get the filter value from the environment variable
 	// filter is for debugging purposes
-	filter := ""
+	err = godotenv.Load("./.env")
+	if err != nil {
+		t.Errorf("Error loading .env file: %v", err)
+	}
+
+	filtersRaw := os.Getenv("INTEGRATION_TESTS_FILTERS")
+	filtersRaw = strings.TrimSpace(filtersRaw)
+	filters := strings.Split(filtersRaw, ",")
```

Comment:
I think it is better for it not to err in case the is no .env and instead defaults to an empty string. 

Also, if it could be nice if I could just could set the env from the console, like

```
INTEGRATION_TEST_FILTERS=alloc make integration
```

Reply:
So is it better to read from the console, instead of .env file?

Reply:
Both would be the ideal solutions. With the console taking priority over the .env file

Reply:
I have achieved that by first checking if the environment variable is set in the console, and if not, then load it from the .env file. @rodrigo-pino